source/databricks/calculation_engine/tests/helpers/delta_table_utils.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pyspark.sql import SparkSession, DataFrame
+from pyspark.sql.types import StructType
+
+
+def write_dataframe_to_table(
+    spark: SparkSession,
+    df: DataFrame,
+    database_name: str,
+    table_name: str,
+    table_location: str,
+    schema: StructType,
+) -> None:
+    spark.sql(f"CREATE DATABASE IF NOT EXISTS {database_name}")
+
+    sql_schema = struct_type_to_sql_schema(schema)
+    spark.sql(
+        f"CREATE TABLE IF NOT EXISTS {database_name}.{table_name} ({sql_schema}) USING DELTA LOCATION '{table_location}'"
+    )
+
+    df.write.format("delta").mode("overwrite").saveAsTable(
+        f"{database_name}.{table_name}"
+    )
+
+
+def struct_type_to_sql_schema(schema: StructType) -> str:
```

Comment:
Perhaps the following? It doesn't seem well suited for general usage in the current form
```suggestion
def _struct_type_to_sql_schema(schema: StructType) -> str:
```